server/app/services/openApi/v2/SecurityType.java

```diff
@@ -0,0 +1,31 @@
+package services.openApi.v2;
+
+/**
+ * Field: type
+ *
+ * <p>Required. The type of the security scheme. Valid values are "basic", "apiKey" or "oauth2".
+ *
+ * <p>https://swagger.io/specification/v2/#security-scheme-object
+ */
+public enum SecurityType {
+  BASIC("basic"),
+  API_KEY("apiKey"),
+//  OAUTH2("oauth2")
```

Comment:
Should we remove this if it's not used, for now?

Reply:
Removed it and just added a note in the comment that it's not supported.